### PR TITLE
Feat/drizzle

### LINKED ISSUE #8: drizzle set up

### PR DESCRIPTION
**Pull request by knhn1004**
resolves #8

### CODE REVIEW

File: db-scripts/createdb.sql [hunk 1]
```diff
@@ -1,4 +1,5 @@
-create database acm_website if not exists;
+SELECT 'CREATE DATABASE acm_website'
```
Comment (knhn1004) [inline db-scripts/createdb.sql:1]: `if not exists` for database wouldn't work

File: db-scripts/createdb.sql [hunk 2]
```diff
@@ -71,7 +73,8 @@ create table if not exists equipment_rentals(
    user_id text,
    date_borrowed date not null default current_date,
    return_date date not null,
-   price money not null,
+--   price money not null,
+   price numeric(10,2) not null,
```
Comment (knhn1004) [inline db-scripts/createdb.sql:77]: `money` type is not compatible with drizzle and it seemed to be more limited, we may also need to update our ER diagram for latest reference

File: server/src/router/v1/v1.router.ts [hunk 3]
```diff
@@ -1,8 +1,24 @@
...
+import { db } from '../../db/db';
+import { users, projects } from '../../db/schema';
+import type { User, Project } from '../../db/schema';
 
 const v1App = new Hono();
 
 // Users
-v1App.get('/users', c => c.json({}));
+v1App.get('/users', async c => {
+	const foundUsers: User[] = await db.select().from(users);
```
Comment (knhn1004) [inline server/src/router/v1/v1.router.ts:10]: some example usage of drizzle orm in our controllers

File: db-scripts/createdb.sql (hunk 2 above)
Comment (ManasC478) [reply db-scripts/createdb.sql:77]: what sort of updates

**Review by ManasC478 — APPROVED** [review]

**Pull request merged**